python/tank/authentication/errors.py

```diff
@@ -71,23 +65,11 @@ def __init__(self, url):
         """
         :param str url: Url of the site where login was attempted.
         """
-        super(ConsoleLoginWithSSONotSupportedError, self).__init__(
+        super(ConsoleLoginNotSupportedError, self).__init__(
             "Authentication using username/password is not supported on "
-            "the console for %s, an SSO-enabled site." % url
```

Comment:
I liked the clarity of SSO/Autodesk Indentity. Maybe pass in the site type in the __init__?

Reply:
My own preferences was to have the exception type directly map to the exception message (meaning if you really need to discriminate, you don't need to do a string search on a potentially localized message).

I've added a description to the message, with the default value of 'Single Sign-On' in case the exception is created using the old name.